themes/classic/templates/catalog/listing/product-list.tpl

```diff
@@ -24,6 +24,10 @@
  *}
 {extends file=$layout}
 
+{block name='hook_extra'}
+  {include file='_partials/microdata-product-list-jsonld.tpl' listing=$listing}
```

Comment:
```suggestion
  {include file='_partials/microdata/product-list-jsonld.tpl' listing=$listing}
```

I recommend you to add a subdirectory for it

Reply:
ok, will do it

Reply:
for the moment I did it only for this issue linked to "product list pages" and use hook_extra from head.tpl to attach it.

Reply:
@PierreRambaud should we use hook_extra or add another block dedicated to JSON LD ? I'm affraid of side effects of using this block

Reply:
It should be a new block

Reply:
done

Reply:
Convinced about the separation I did =>

- New file for specific product microdata (ref in product.tpl)
- Remove meta from head microdata file => head.tpl for global meta, product.tpl for product ones